plugins/janus_streaming.c

```diff
@@ -6519,6 +6521,23 @@ static int janus_streaming_rtsp_connect_to_server(janus_streaming_mountpoint *mp
 	}
 	long code = 0;
 	res = curl_easy_getinfo(curl, CURLINFO_RESPONSE_CODE, &code);
+#if CURL_AT_LEAST_VERSION(7, 62, 0)
+	if(code == 404) { //Possibly a quirk in the RTSP server, where the DESCRIBE request expects a path only.
+		CURLU *curl_u;
+		char *path;
+		curl_u = curl_url();
+		if(!(curl_url_set(curl_u, CURLUPART_URL, source->rtsp_url, 0))) {
+			if(!(curl_url_get(curl_u, CURLUPART_PATH, &path, 0))) {
+				curl_easy_setopt(curl, CURLOPT_RTSP_STREAM_URI, path);
+				res = curl_easy_perform(curl);
+                                source->rtsp_stream_uri = path;
```

Comment:
Indentation broken here (spaces instead of tabs). That said, I'd do a `g_strdup()` for `rtsp_stream_uri`, so you can use the `curl_free(path)` you commented below.